packages/video_player/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+## 1.0.0
+
+* Initial release.
+
+## 2.0.0
+
+* Update Dart and Flutter SDK constraints
+* Update Flutter copyright information
+* Update example and integration_test
+* Update platform interface to 2.0.1
```

Comment:
The platform interface version is 4.1.0.

(Actually we don't need the changelog for "2.0.0" because the 1.0.0 version has not been released and technically you are adding this package for "initial release".)

Reply:
I changed the version to 1.0

Reply:
Thanks!